symfony/templates/_menu_item.html.twig

```diff
@@ -1,5 +1,14 @@
 {% set is_active = app.request.attributes.get('_route') == name %}
 
-<li class="nav-item">
-    <a class="nav-link {% if is_active %}active{% endif %}" {% if is_active %}aria-current="page"{% endif %} href="{{ path(name) }}">{{ pretty_name|raw }}</a>
-</li>
+<a
+    {% if priority is defined %}data-priority="{{ priority }}"{% endif %}
+    class="nav-link{% if is_active %} active{% endif %}"
+    {% if is_active %}aria-current="page"{% endif %}
+    href="{{ path(name) }}"
+>
+    <i class="fas fa-{{ fa_icon }}"></i>
+    <span class="label">
+        <div class="spacer">{{ label }}</div>
+        <div class="displayed">{{ label }}</div>
```

Comment:
Why is the label doubled with different classes?

Reply:
In 959596cfff8dec6019e305bf222cc945435f9c28, I make it so the active page in the navbar will have its text bolded.

Bolding text makes it wider:
> Fursuit Makers
> **Fursuit Makers**

This made the navbar items "move around" a little bit. There's no standard math formula, but generally, shorter text will have less difference in width between unbolded and bolded. So if you were on a page with a short label (eg "Contact") and went to a page with a long label (eg "Fursuit Makers"), the Contact label might get 8px narrower but Fursuit Makers would get 18px wider — so 10px net increase.

I disabled the code we're talking about in this comment, and then took these two pictures. These screenshots were taken at the same browser width. All that changed is which page is active:
<img width="1018" alt="Screenshot 2024-09-07 at 9 59 49 AM" src="https://github.com/user-attachments/assets/971f3268-6fed-4258-8c0d-0528254778d1">
<img width="1019" alt="Screenshot 2024-09-07 at 9 59 57 AM" src="https://github.com/user-attachments/assets/1e29aeaa-179b-4587-8a60-cddf512d3953">
The net width increase was enough that it pushed an item into the overflow menu.

_Overall_, this difference is not that big usually. But it was enough that while I was testing, it felt like the navbar changed when I went to a new page.

So: I have two `<div />`s with the labels. The `.displayed` node is what we show the user. When the page isn't active, it's unbolded; when it's active, it is bolded. Meanwhile, the `.spacer` node is invisible — it has `opacity: 0`. But it is permanently bolded. I do this so that the overall bounding box of the `.nav-link` reserves the full space it would need when the link is bolded. This means that the width of each nav item becomes stable.